src/pages/[...slug].tsx

```diff
@@ -189,20 +159,29 @@ const StyledIcon = styled(FontAwesomeIcon)`
 // -> You can not use getInitialProps with getServerSideProps. Please remove getInitialProps. /[...slug]
 
 // TODO: needs type declaration
-export const getServerSideProps: GetServerSideProps<any, any> = async (
-  props
-) => {
+export const getServerSideProps: GetServerSideProps = async (props) => {
```

Comment:
This still needs a type, right? Because `props` is implicitly `any` here when you don't specify the type arguments in `GetServerSideProps`.

Reply:
currently props has this type:
```
(parameter) props: GetServerSidePropsContext<ParsedUrlQuery>
```
not sure.